helm-chart/binderhub/values.yaml

```diff
@@ -35,6 +35,18 @@ github:
   # ...or access_token
   accessToken:
 
+# have to set cors.allowOrigin twice:
+# once in top-level cors.allowOrigin,
+# and again in jupyterhub.hub.extraConfigMap.cors.allowOrigin
+
+# &cors anchor allows us to remove redundancy in this values.yaml,
```

Comment:
Perhaps:

```
# Using YAML anchors, `&cors` for the first appearance and `*cors` for subsequent
# appearances, allows us to remove redundancy in this (BinderHub) `values.yaml`.
# The anchors do not extend beyond this file.
# As such, users must set `cors` separately for their own user `values.yaml` for notebooks. 
# `cors` will be set separately in the user `values.yaml` and binderhub`values.yaml`.
# The same anchor pattern (`&cors`, `*cors`) can be used in the user `values.yaml`.
```

Reply:
@minrk A bit more context above on YAML anchors will make the pattern more understandable.

Reply:
Thanks! added your comment.